dagger/run_dbt_spark_tests.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+import argparse
+import sys
+
+import anyio as anyio
+import dagger as dagger
+from dotenv import find_dotenv, load_dotenv
+
+PG_PORT = 5432
+load_dotenv(find_dotenv("test.env"))
+# if env vars aren't specified in test.env (i.e. in github actions worker), use the ones from the host
+TESTING_ENV_VARS = {env_name: os.environ[env_name] for env_name in os.environ
+                    if env_name.startswith(("DD_", "DBT_"))}
+
+TESTING_ENV_VARS.update({
+    "ODBC_DRIVER": "/opt/simba/spark/lib/64/libsparkodbc_sb64.so",
```

Comment:
this is set here and not in the workflow since it's defined by the configure_odbc.sh script